tests/operator/operator.go

```diff
@@ -2517,13 +2546,14 @@ spec:
 		})
 
 		AfterEach(func() {
-			// Reset the feature gate to its original value after running any of these tests.
-			if fgEnabled {
-				tests.EnableFeatureGate(virtconfig.CommonInstancetypesDeploymentGate)
-			} else {
-				tests.DisableFeatureGate(virtconfig.CommonInstancetypesDeploymentGate)
+			// Do nothing if the current config already matches the original
+			kv := libkubevirt.GetCurrentKv(virtClient)
+			if reflect.DeepEqual(originalConfig, kv.Spec.Configuration.CommonInstancetypesDeployment) {
```

Comment:
We should only test a default behavior and unit test (empty struct, false and true). Alternatively test one negative case. WDYT?

Reply:
Was this comment against an older revision with multiple tests asserting the behaviour of the feature configurable? It's a single test in the latest revision now and IMHO it's cheap enough to test the behaviour in this single context than just default and limited negative testing.

Reply:
Ah sorry it was against the commit, GH's UX is awful sometime.

Anyway my point remains, is this still an issue with a single test asserting this behaviour?

Reply:
Somehow I missed the last commit... Sorry for that..